pkg/workloads/tf_api/api.py

```diff
@@ -380,6 +380,21 @@ def predict(deployment_name, api_name):
     return jsonify(response)
 
 
+def valid_model_dir(model_dir):
+    """
+    validates that model_dir has the expected directory tree, e.g:
+
+    {TF serving version number}/
+        saved_model.pb
+        variables/
+            variables.data-00000-of-00001
+            variables.index
+    """
+    version = os.listdir(model_dir)[0]
```

Comment:
Have you confirmed that it must be a digit? Is that a TF Serving thing?

Reply:
yes